presto-main/src/main/java/com/facebook/presto/util/MergeSortedPages.java

```diff
@@ -44,6 +44,12 @@ private MergeSortedPages() {}
             AggregatedMemoryContext aggregatedMemoryContext,
             DriverYieldSignal yieldSignal)
     {
+        requireNonNull(pageProducers, "pageProducers is null");
+        requireNonNull(comparator, "comparator is null");
+        requireNonNull(outputTypes, "outputTypes is null");
+        requireNonNull(aggregatedMemoryContext, "aggregatedMemoryContext is null");
+        requireNonNull(yieldSignal, "yieldSignal is null");
```

Comment:
This these things are passed through to `mergeSortedPages`, it's redundant to check them here.